internal/pkg/cli/env_init.go

```diff
@@ -470,7 +470,6 @@ func (o *initEnvOpts) deployEnv(app *config.Application) error {
 		Name:                     o.Name,
 		AppName:                  o.AppName(),
 		Prod:                     o.IsProduction,
-		PublicLoadBalancer:       true, // TODO: configure this based on user input or service Type needs?
```

Comment:
Why do we want to remove this one? Are we planning to substitute it?

Reply:
It's because it's not used anywhere in the template :D we already set a default for that parameter in the legacy template ("true") so the LB is always created even if the parameter is not provided